app-dicts/myspell-en/myspell-en-20230101.ebuild

```
@@ -0,0 +1,57 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit myspell-r2
+
+DESCRIPTION="English dictionaries for myspell/hunspell"
+HOMEPAGE="
+	https://extensions.libreoffice.org/extensions/english-dictionaries
+	https://proofingtoolgui.org
+	https://github.com/marcoagpinto/aoo-mozilla-en-dict
+"
+SRC_URI="https://extensions.libreoffice.org/assets/downloads/41/1672590731/dict-en-20230101_lo.oxt"
+
+LICENSE="BSD MIT LGPL-3+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~sparc-solaris ~x86-solaris"
+
+PLOCALES=( "en-AU" "en-CA" "en-GB" "en-US" "en-ZA" )
+IUSE+="+l10n_en ${PLOCALES[@]/#/l10n_}"
+REQUIRED_USE="|| ( l10n_en ${IUSE[@]} )"
```

```suggestion
REQUIRED_USE="|| ( l10n_en ${PLOCALES[@]/#/l10n_} )"
```
Using IUSE in REQUIRED_USE expands to something like `"|| ( l10n_en +l10n_en ... "`, which is invalid.

You can install `pkgcheck` and run `pkgcheck scan` before committing, it validates the ebuild.

---

fixed.